### PR TITLE
Prune deleted products, don't reset indices

### PR DESCRIPTION
**Pull request by masonmcelvain**
Closes https://github.com/iFixit/ifixit/issues/54409

Resetting seems to make Akeneo unsearchable until we rebuild the indices. Let's try pruning deleted products/product models before rebuilding the indices, which should have the same effect.

CC @danielbeardsley 

## On deploy

Update the Akeneo crontab to use `reindex-in-place` instead of `reindex`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -106,7 +106,7 @@ bootstrap-database:
 
 .PHONY: reindex
 reindex:
-	$(CONSOLE) akeneo:elasticsearch:reset-indexes
```
Comment (danielbeardsley) [inline Makefile:109]: Great! I think this was in here cause this was part of the *install akeneo* process (which does need to re-create them).

I think we should probably have another Make target that resets the indexes first and then does this `rebuild_indexes` or some such. That should be triggered wherever this is currently triggered in the makefile.

**Review by danielbeardsley — DISMISSED** [review]
Good job finding that!

dev_block :+1: on the comment

File: Makefile (hunk 1 above)
Comment (masonmcelvain) [reply Makefile:109]: Thanks for the feedback! Maybe we could leave the current Make target the same and add a new one that we can reference in the crontab. Something like:

```make
.PHONY: reindex
reindex:
	$(CONSOLE) akeneo:elasticsearch:reset-indexes
	$(MAKE) reindex-in-place

.PHONY: reindex-in-place
reindex-in-place:
	$(CONSOLE) pim:product:clean-removed-products
	$(CONSOLE) pim:product-model:index --all
	$(CONSOLE) pim:product:index --all
```

Comment (danielbeardsley) [reply Makefile:109]: Perfect!

**Comment by masonmcelvain** [comment]
un_dev_block 👍🏻

**Comment by danielbeardsley** [comment]
CR :+1: 

We'll also need a pull on server-templates to have the cron job call this new target instead of `reindex`

**Pull request merged**